### PR TITLE
Add low disk space on crawler

### PR DESCRIPTION
**Pull request by smford**
This details the manual fix for low disk space alerts on
/mnt/crawler_worker

### CODE REVIEW

File: source/manual/alerts/low-available-disk-space.html.md [hunk 1]
```diff
@@ -59,6 +59,35 @@ This command will take a while to complete, depending - of course - on
 how large your two starting folders are. After it has completed, push
 Nagios to re-check the service on each affected host.
 
+## Low available disc space on /mnt/crawler_worker on mirrorer
+
+This can be caused when the AWS autoscaler re-spins a VM and the newly created
+VM does not attach the secondary EBS volume properly.
```
Comment (benthorner) [inline source/manual/alerts/low-available-disk-space.html.md:65]:
```suggestion
VM does not attach the secondary EBS volume properly. In this case, the alert will say `DISK CRITICAL: /mnt/crawler_worker not found`.
```

File: source/manual/alerts/low-available-disk-space.html.md [hunk 2]
```diff
@@ -59,6 +59,35 @@ This command will take a while to complete, depending - of course - on
...
+the same time also spinning up the replacement VM.  The new VM can spin up and be
+provisioned faster than the lock can be released.
+
+### Solution
+Manually re-attach the EBS volume tagged with mirrorer to the mirrorer
+instance, then run puppet on the instance. This will cause puppet to correctly
+mount the EBS volume in /mnt/crawler_worker.
+
+1. Login to https://gds-users.signin.aws.amazon.com/console
+1. Switch role to the appropriate environment: Integration, Staging or Production
```
Comment (benthorner) [inline source/manual/alerts/low-available-disk-space.html.md:79]: We can replace these first two steps with `gds aws govuk-integration-powerusers -l` (as an example). We try to use GDS CLI wherever possible, so there's one clear way of doing things.

File: source/manual/alerts/low-available-disk-space.html.md [hunk 3]
```diff
@@ -59,6 +59,35 @@ This command will take a while to complete, depending - of course - on
...
+1. Click Services -> EC2
+1. Change region to "Europe (Ireland) eu-west-1"
```
Comment (benthorner) [inline source/manual/alerts/low-available-disk-space.html.md:81]: Interesting: do you need to do this? (I've never had to do this.)

File: source/manual/alerts/low-available-disk-space.html.md [hunk 4]
```diff
@@ -59,6 +59,35 @@ This command will take a while to complete, depending - of course - on
...
+1. Click Instances on the left hand side menu bar and filter by "mirrorer"
+1. Copy the Instance ID, it should look similar to `i-06887a467af1266a0`
+1. Left hand side menu, expand Elastic Block Store, and click Volumes
+1. Filter by "mirrorer" and ensure that the state of the only result is "available"
+1. Right click volume and choose "Attach Volume"
+1. Paste the Instance ID in to the appropriate box, leaving Device as the default
+1. Click Attach
+1. After AWS finishes attaching the volume to the instance the state should change to "in-use", once it has settled on that state SSH in to the mirrorer instance and run `sudo puppet agent --test`
```
Comment (benthorner) [inline source/manual/alerts/low-available-disk-space.html.md:89]: I'd suggest splitting the last part into a new step or paragraph. 

Minor: I think we usually run `govuk_puppet --test`, so we should probably say that. On the other hand, we seem to have very little clear documentation about running our weird Puppet snowflake!

**Review by benthorner — COMMENTED** [review]
Thanks @smford! While it's shame this happens, now 2ndline can solve this for themselves and leave you in peace 🎉. I've left a few minor comments; the one about logging in is more major, since the steps should follow [our standard approach](https://docs.publishing.service.gov.uk/manual/get-started.html#8-use-your-aws-access).

File: source/manual/alerts/low-available-disk-space.html.md (hunk 3 above)
Comment (smford) [reply source/manual/alerts/low-available-disk-space.html.md:83]: It depends on the user and whether they use regions.

File: source/manual/alerts/low-available-disk-space.html.md (hunk 4 above)
Comment (smford) [reply source/manual/alerts/low-available-disk-space.html.md:91]: 👍

File: source/manual/alerts/low-available-disk-space.html.md (hunk 2 above)
Comment (smford) [reply source/manual/alerts/low-available-disk-space.html.md:81]: 👍

**Review by benthorner — APPROVED** [review]
Looks good - thanks again @smford 👍

**Pull request merged**